Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Hammer of twilight

## Changelog:
- [x] Added Hammer of Twilight artifact.

- [x] Added icon for it

- [ ] Removed the event that adds it to onyxia for testing purposes

Balancing Spreadsheet:
https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=1165316868

# How to test:
Select Onyxia that rules the duchy of Wyrmborg and check that she has the artifact
